### PR TITLE
fix(SlideToggle): Refactor component

### PR DESCRIPTION
**Pull request by patrickliu-sa**
Graduation of the component from SEEK one to the version that we customized for SEEK Asia

**Spec for SEEK Asia version:**
Screenshot 2019-04-16 at 9 56 30 AM

I've also added back `aria-checked` and `tabindex="0"` attributes for better accessibility

### CODE REVIEW

File: react/SlideToggle/SlideToggle.js [hunk 1]
```diff
@@ -10,65 +9,53 @@ const RIGHT = 'right';
...
-    checked: PropTypes.bool,
-    className: PropTypes.string
+    rtl: PropTypes.bool
   }
 
   static defaultProps = {
+    hideLabel: false,
     position: RIGHT,
-    hideLabel: false
+    rtl: false
```
Comment (mengtzu) [inline react/SlideToggle/SlideToggle.js]: No need to shorten this, readable is better `rightToLeft`?

**Comment by mengtzu** [comment]
Checking the preview, looks like it doesn't work when the label is on the left

File: react/SlideToggle/SlideToggle.js (hunk 1 above)
Comment (patrickliu-sa) [reply react/SlideToggle/SlideToggle.js]: Agree. Just renamed it as suggest. 😸

**Comment by patrickliu-sa** [comment]
> Checking the preview, looks like it doesn't work when the label is on the left

The old tactics is pop and push label before or after the handle, but I let css do their job solely.

File: react/SlideToggle/SlideToggle.less [hunk 2]
```diff
@@ -1,109 +1,104 @@
...
-}
-
-.labelText {
-  line-height: @grid-row-height * 6;
-}
-
 .slider {
-  position: relative;
+  background-color: @saGrey4;
+  border-radius: @grid-row-height * 3;
```
Comment (ChanJit) [inline react/SlideToggle/SlideToggle.less]: just wondering should radius have their own unit? instead of using row-height as their unit base?

Comment (patrickliu-sa) [reply react/SlideToggle/SlideToggle.less]: I've replaced it with a self explanatory formula as the value now.

**Pull request merged**